openupgradelib/openupgrade_merge_records.py

```diff
@@ -861,7 +864,10 @@ def _delete_records_sql(
     env, model_name, record_ids, target_record_id, model_table=None
 ):
     if not model_table:
-        model_table = env[model_name]._table
+        try:
```

Comment:
Not sure about all of this new `try`. If the model is not available, another bigger problem may arise, so I'm thinking that better to detect it sooner than later.

Reply:
This is `_delete_records_sql()`, which used only when method='sql'. My point is we should be able to use the merge records with method='sql' in tables that are obsolete or not currently in the env in the moment the merge is called.